README.md

```diff
@@ -12,4 +12,47 @@ Please find the sample that fits your use-case from the table below.
 | [Custom Login Page](/custom-login) | A React application that uses the Okta Sign-In Widget within the React application to authenticate the user. |
 
 
-[Okta React Library]: https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react
\ No newline at end of file
+[Okta React Library]: https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react
+
+## Running E2E Tests locally
+
+E2E Tests will be run against the Custom Login and Okta-Hosted Login servers
+
+Before running the tests locally, install all the dependencies 
```

Comment:
Let's write this as "In addition to running `npm install` in the root of the project (to install the dev dependencies for testing) you will also need to install the dependencies of each library:"